### DISCUSSION TITLE
Data Logging?

### DISCUSSION THREAD

**Discussion by ElliottS200** [security-mountain/security-mountain.github.io#10] [Ideas]
This is a tricky one, as some users may not see the point and skip it, but how do people feel about keeping logs of important data and whether it is secured sufficiently? Would you bother doing this personally?

**Comment by apollodelphi702** [comment]
Perhaps, as mentioned previously we could organize tips by 'level of paranoia'. I'd say security is always about threat modelling - some users who don't use technology often or for very much won't have to worry about people who essentially 'live online', certainly I could imagine people who are operating at a higher 'level of paranoia' would be interested in this.
